src/components/create-safe/status/hooks/useWatchSafeCreation.ts

```diff
@@ -35,8 +38,21 @@ const useWatchSafeCreation = ({
 
     if (status === SafeCreationStatus.INDEXED) {
       trackEvent(CREATE_SAFE_EVENTS.GET_STARTED)
+      const chainPrefix = chain?.shortName
```

Comment:
I added the chain prefix to the redirect to fix a bug where if you refresh the app using an URL with a non-EIP3770 address the app shows a 404 error